app/frontend/Calendar/EditEvent/DialogHeader.svelte

```diff
@@ -168,6 +168,11 @@
 
   $: event.startEditing(); // not `$event`
   let newCalendar = event.calendar; // Don't listen to $event #730
+  let prevEventID = event.id;
+  $: if (event.id != prevEventID) {
+    newCalendar = event.calendar;
+    prevEventID = event.id;
+  };
```

Comment:
Write multiple lines as function, like so:

```suggestion
  $: updateEvent(event);
  let prevEventID = event.id;
  function updateEvent(event) {
    if (event.id == prevEventID) {
      return;
    }
    prevEventID = event.id;

    newCalendar = event.calendar;
  };
```

Reply:
done